src/AlohaKit/Controls/LinearGauge/LinearGauge.cs

```diff
@@ -20,7 +20,7 @@ public LinearGauge()
         public LinearGaugeDrawable LinearGaugeDrawable { get; set; }
        
         public static readonly new BindableProperty BackgroundProperty =  
-            BindableProperty.Create(nameof(Background), typeof(Brush), typeof(Button), null,
+            BindableProperty.Create(nameof(Background), typeof(Brush), typeof(Button), new SolidColorBrush(Colors.Pink),
```

Comment:
Could you create a static class where define the Controls Themes with colors etc?